src/Events/DispatchAudit.php

```diff
@@ -22,4 +23,88 @@ public function __construct(Auditable $model)
     {
         $this->model = $model;
     }
+
+    /**
+     * Prepare the instance values for serialization.
+     *
+     * @return array
+     */
+    public function __serialize()
+    {
+        $values = [
+            'class' => get_class($this->model),
+            'model_data' => [
+                'exists' => true,
```

Comment:
You should also save the model id, so that it can be retrieved by id in unserialization

Reply:
I see

Reply:
Also Id comes with attributes, original properties